src/librustc_codegen_ssa/mir/analyze.rs

```diff
@@ -182,23 +191,48 @@ impl<'mir, 'a: 'mir, 'tcx: 'a, Bx: BuilderMethods<'a, 'tcx>> Visitor<'tcx>
                         // Recurse with the same context, instead of `Projection`,
```

Comment:
This comment needs updating or should be removed.

Reply:
There is still a recursive call to `process_place` (sadly), so this appears to still be correct.